### PR TITLE
fix "az sql server firewall-rule update" example

### PR DESCRIPTION
**Pull request by Linaa**
end-ip-address must be larger than start-ip-address.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@MyronFanQiu  please deal with this PR.

File: src/azure-cli/azure/cli/command_modules/sql/_help.py [hunk 1]
```diff
@@ -641,7 +641,7 @@
 short-summary: Update a firewall rule.
 examples:
   - name: Update a firewall rule
-    text: az sql server firewall-rule update -g mygroup -s myserver -n myrule --start-ip-address 9.8.7.6 --end-ip-address 5.4.3.2
+    text: az sql server firewall-rule update -g mygroup -s myserver -n myrule --start-ip-address 5.4.3.2 --end-ip-address 9.8.7.6
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/sql/_help.py]: Just one small question. I'm not familiar with sql. What's the reason for this PR?

Comment (MyronFanQiu) [reply src/azure-cli/azure/cli/command_modules/sql/_help.py]: oh. I got the point.

**Comment by MyronFanQiu** [comment]
Thanks for your contribution.

**Pull request merged**